packages/integration-karma/test/light-dom/multiple-templates/index.spec.js

```diff
@@ -23,7 +23,9 @@ describe('multiple templates', () => {
         element.next();
         return Promise.resolve().then(() => {
             expect(element.querySelector('div').textContent).toEqual('b');
-            expect(getComputedStyle(element.querySelector('div')).color).toEqual('rgb(0, 0, 0)');
+            expect(getComputedStyle(element.querySelector('div')).color).toEqual(
+                'rgb(233, 150, 122)'
+            );
```

Comment:
This is the thing that changed in light DOM. Basically the styles from template A remain even when template B is rendered.

I don't think this is a big deal, because it's already how synthetic shadow works – styles at the global `<head>` are never cleaned up. [Many CSS-in-JS libraries work this way.](https://twitter.com/sebmarkbage/status/1422278240866078725)

Reply:
I opened a separate issue for this; it's a general problem: https://github.com/salesforce/lwc/issues/2466